src/components/thumbnail-card/ThumbnailCardDetails.js

```diff
@@ -2,17 +2,21 @@
 import * as React from 'react';
 
 type Props = {
+    actionItem?: React.Node,
     icon?: React.Node,
     subtitle?: React.Node,
     title: React.Node,
 };
 
-const ThumbnailCardDetails = ({ icon, subtitle, title }: Props) => (
+const ThumbnailCardDetails = ({ actionItem, icon, subtitle, title }: Props) => (
     <div className="thumbnail-card-details">
         {icon}
         <div className="thumbnail-card-details-content">
-            <div className="thumbnail-card-title">{title}</div>
-            {subtitle && <div className="thumbnail-card-subtitle">{subtitle}</div>}
+            <div className="thumbnail-card-details-body">
```

Comment:
Open to suggestions for the naming of this div. I was trying to create a separation between the title + subtitle and the actionItem. Body -> title + subtitle

Reply:
New CSS classes should use the SUIT naming convention: https://github.com/suitcss/suit/blob/master/doc/naming-conventions.md. So, this could be `ThumbnailCardDetails-bodyText`.

Reply:
Thanks for the callout!